applications/tari_validator_node/src/base_layer_scanner.rs

```diff
@@ -267,8 +282,8 @@ impl BaseLayerScanner {
             for output in utxos.outputs {
                 let output_hash = output.hash();
                 if output.is_burned() {
-                    warn!(target: LOG_TARGET, "Burned output encountered. Not yet implemented");
-                    // self.register_burnt_utxo(output.commitment);
+                    info!(target: LOG_TARGET, "Found burned output: {}", output_hash);
+                    self.register_burnt_utxo(output.commitment)?;
```

Comment:
Maybe only the shards @ H(commitment) should save this substate, so that it can be DOWNed on claim